Update readme.md

AfubeAngel opened the pull request:
# Pull Request Template

## What
Updated the README.md and CONTRIBUTING.md files to enhance clarity and guidance for new contributors.

## How
Revised sections for installation and contribution guidelines to improve usability and understanding. Added detailed instructions on setting up the development environment and contributing effectively.

## Additional Notes
These updates aim to streamline the onboarding process for new contributors and improve overall project documentation.

issue #94

AfubeAngel commented:
@manuelalferez Please assist with the authorization for the deployment. I updated the CONTRIBUTING.md as directed and included the changes you made in the README.md

manuelalferez commented on CONTRIBUTING.md:
```diff
@@ -1,10 +1,70 @@
+# Contributing to Flowinance
 
-When contributing to this repository, please first discuss the change you wish to make [**creating an issue**](https://github.com/manuelalferez/flowinance/issues).
+We’re so glad you’re interested in contributing to our **open-source project**! Before getting started, please take a moment to read our guidelines to ensure a smooth and efficient contribution process.
```
```suggestion
We are glad that you are interested in contributing to our open source project!
```

manuelalferez commented on CONTRIBUTING.md:
````diff
@@ -1,10 +1,70 @@
+git clone https://github.com/[your-github-username]/flowinance.git
+```
+
+### 3. Navigate to the directory
+``` bash
+cd flowinance
+```
+### 4. Create a new branch for your feature/bugfix:
+```bash
+git checkout -b your-branch-name
+```
+
+### 5. Stage and Commit your changes:
+```bash
+git add .
+```
+
+```bash
+git commit -m "Add your message here"
+```
+
+### 6. Push your changes to GitHub:
+```bash
+git push origin your-branch-name
+```
+
+### 7. Create a pull request and describe the changes.
+ Please refer to the [pull request format](https://github.com/manuelalferez/flowinance/blob/main/.github/PULL_REQUEST_TEMPLATE.md)
+
+ <br>
+
+## 💬 Have feedback or questions? Join the conversation!
+
+We encourage you to visit our [Flowinance Discussions](https://github.com/manuelalferez/flowinance/discussions) for general questions, feature suggestions, or to engage with the community. It's the best place to share ideas and collaborate!
+
+## Ask me 🤙
+
+You can always contact me via [Telegram](https://t.me/manuelalferez) if you want to talk more about the project 😊
+
+Thank you for your contributions! Together, we can make Flowinance even better 🎉
````
```suggestion
```

manuelalferez replied:
We make contributors read a lot: we will respect everyone's time.

AfubeAngel replied:
@manuelalferez Apologies. I wanted to break it down for first-time contributors, I guess it isn't necessary. Apologies again

manuelalferez commented on CONTRIBUTING.md:
```diff
@@ -1,10 +1,70 @@
+1. Discuss Changes Before Starting
+Please first discuss the change you wish to make [**creating an issue**](https://github.com/manuelalferez/flowinance/issues).
 
-1. Make sure the change runs successfully locally
-2. Consider if the change affects `README.md`
-3. In your commit message's subject (50 char length at max) you should succinctly describe what changes this commit introduces
-4. A commit shouldn't take more than 100 lines of code. You will write your contribution faster and the person who will review your code will appreciate it.
```
Please, keep this.

AfubeAngel commented:
@manuelalferez Is it alright with you if I  close this pull request and open a new one considering all the corrections? As you mentioned, the README.md is fine. So I added the table of contents for faster navigation and updated the prerequisites to take supabase into account. I apologize for the duplication. The primary changes were done in the CONTRIBUTING.md which I had to write with fresh contributors or beginners in mind but since it is too long; I would shorten it according to your directions.

Or should I go ahead and make the changes on this pull request? Thank you

manuelalferez commented:
@AfubeAngel It is better to keep everything in the same PR, this way the whole history of the improvement is kept in a single point and not in multiple places. But as it is easier for you, if you see that it is going to be too much, open a new one 😊

AfubeAngel commented:
@manuelalferez I'll just push on this PR.

Please is this shorter version acceptable for the contributing.md?
![image](https://github.com/user-attachments/assets/95fac03b-cb2e-4672-a44d-3b76ea04e44b)

manuelalferez commented:
@AfubeAngel The guideline is already an explanation of how to contribute, the how to contribute only explains how to install the project: that information is already in the README. maybe you can change it to something like: Installation, 
“Follow the instructions in the readme to install and run the project” and add a link

manuelalferez commented:
@AfubeAngel great job!! Much better. Ready to merge
